### DISCUSSION TITLE
Filebase, don't see files after scan

### DISCUSSION THREAD

**Discussion by marralser** [NuSkooler/enigma-bbs#567] [Q&A]
Hi Folks,

it may be a simple one, but I fail to fo find the solution. I configured a file base, by both defining the storage tags and areas.. The areas then show up when I log in to the BBS filebase. However, when I copy files into the respective paths, and then run a filebase scan with  e.g. ./oputil.js  fb scan --quick --tags primiary_research (I also tried without defining tags or quick), I can't find any files in the BBS system. The log files does not show anything suspicious. Is there any additional conditions the files have to meet in order to be detected? Possilby I misunderstood the concept! 

 My config looks like this
```
storageTags: {
        primary_research: "/home/markus/enigma-bbs/file_base/primary_research"
    }


areas: {
        //
        //  Area for Primary Research Articles
        //
        primary_research: {
            name: Primary Research Articles
            desc: Collection of our primary research articles.
            storageTags: [
                "primary_research", "articles", "research"
            ]
        }
```
Any suggestion welcome!

**Comment by NuSkooler** [comment]
@marralser 

Can you provide the output of the following command?

```bash
./oputil.js fb info primary_research
```

What you should see is a quick dump of info on the `primary_research` area. If not, there is something up with the configuration -- perhaps the fragments examples aren't in the right spots? Does the research group show up in the area list on the board?

**Comment by marralser** [comment]
Thanks Bryan for looking at this! Here is the output 

./oputil.js fb info primary_research
areaTag: primary_research
name: Primary Research Articles
desc: Collection of our primary research articles.
storageTag: primary_research => /home/markus/enigma-bbs/file_base/primary_research

**Reply by NuSkooler** [reply]
Thanks, that definitely looks correct, assuming the path displayed looks right to you.

Do you get interesting output when attempting to scan the area? It may be a typo in your command above: it looks like `--tags` is receiving "primary_research", but no area is supplied. Can you try the following?

```bash
./oputil.js fb scan --tags "research" primary_research
```

Where the tags value you can put any comma separated tags you like.

**Reply by marralser** [reply]
Thanks it worked seemlessly! Maybe this could help others too, i,.e. you could update the documentationaccordingly?

**Reply by marralser** [reply]
I hope you don't mind a second question to the filebase? I was looking for SEXYZ linux binaries as you mentioned them in your very nice and exitensive github documentation (https://nuskooler.github.io/enigma-bbs/configuration/file-transfer-protocols.html), but after I couldn't find them, I  noticed a binary of same name in the docker directory  (/enigma-bbs/docker/bin/sexyz). I guess the docker bin might not include the modem transfoer protocols, no? If no, would you perhaps be able to share a download location for the linux binaries ? Highly appreciated!

**Reply by NuSkooler** [reply]
The binary in the Docker dir should be good as long as it matches your system's architecture et al., most likely it will work for you.
